docker-compose.yml

```
@@ -1,28 +1,28 @@
 version: "3.7"
 
 services:
-
   backend:
     build: .
     image: tender/backend:latest
     ports:
       - "3333:3333"
-#    depends_on:
-#      - db
+    depends_on:
+      - neo4j_db
+    networks:
+      - tender-net
+
+  neo4j_db:
+    image: library/neo4j:latest
```

Ist zwar jetzt für unser Projekt wahrscheinlich nicht so wichtig, aber allgemein sollte lieber ne bestimmte Version festgelegt werden.
```suggestion
    image: neo4j:4.2
```